readme.txt

```diff
@@ -380,1167 +380,7 @@ The 2% fee on Stripe donations only applies to donations taken via our free Stri
-* New: The email system within GiveWP has been revamped so it's easier to toggle on or off and customize the various notifications all via one interface. [#1346](https://github.com/impress-org/givewp/issues/1346)
-* New: Donors can now have multiple addresses attached to their profiles. This is helpful in the case of donors having a separate billing and mailing address and also to segment other addresses like Gift Aid addresses, etc. [#370](https://github.com/impress-org/givewp/issues/370)
-* New: The tooltip library being used has been switched from qTip2.js to Hint.css. Not only do the new tooltips look nicer, they are also more light weight and have better compatibility with the various WP themes. [#619](https://github.com/impress-org/givewp/issues/619)
-* New: You can now set donation goals based on total number of donations. For instance, setting a goal to accept 500 total donations is now possible. This is useful if you don't want to show amount or percentage based goals. [#1443](https://github.com/impress-org/givewp/issues/1443)
-* New: Revamped how donors can create an account when giving. Rather than adding additional form fields there's now a single "Create an account" checkbox. If checked upon submission of the donation a new WP User will be automatically created for that donor. [#1517](https://github.com/impress-org/givewp/issues/1517)
-* New: Implementing query caching in wp-admin for intense queries. You should see improved load times now when viewing your donations, donors, reports and more. [#1944](https://github.com/impress-org/givewp/issues/1944), [#2121](https://github.com/impress-org/givewp/issues/2121)
-* New: The upgrade process now works in the background and is much more reliable on various server environments. You can also view overall progress in percentage format and see a stepped progress bar on the upgrade screen. [#2531](https://github.com/impress-org/givewp/issues/2531)
-* Tweak: The plugin's images are now minified for faster load times. [#1035](https://github.com/impress-org/givewp/issues/1035)
-* Tweak: Renamed the give_customers and give_customermeta tables to give_donors and give_donormeta. [#1960](https://github.com/impress-org/givewp/issues/1960)
-* Tweak: Added a nonce verification to donation forms for additional security. [#2568](https://github.com/impress-org/givewp/issues/2568)
-* Fix: Sorting by donation statuses wasn't returning the expected results in wp-admin. [#2400](https://github.com/impress-org/givewp/issues/2400)
-* Fix: Donors are now required to actually add content into the First Name field and not just use a space character to pass the validation. [#2562](https://github.com/impress-org/givewp/issues/2562)
+= [See GiveWP's full changelog on Github](https://github.com/impress-org/givewp/blob/master/changelog.txt) =
```

Comment:
Slight wording suggestion change to avoid the apostrophe and looks like the heading doesn't play nice with the markdown link

```suggestion
[See the full GiveWP changelog on Github](https://github.com/impress-org/givewp/blob/master/changelog.txt)
```